archivers/bzip3/Portfile

```
@@ -0,0 +1,22 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        kspalaiologos bzip3 1.2.1
+github.tarball_from releases
+categories          archivers
+license             LGPL-3
+
+maintainers         {@sech1p gmail.com:sech1piam} openmainter
+
+description         A better and stronger spiritual successor to BZip2.
+long_description    {*}${description}
+
+homepage            https://github.com/kspalaiologos/bzip3
+
+platforms           darwin
+
+checksums           rmd160  3940e049433a348ccc9fe4f0fcc3fe76cbae37ad \
+                    sha256  bba90d867f53efa4291de1df9c22da1578e1a93ca819e0fc68881da6fcc4149d \
+                    size    399621
```

The build log does not show the commands that were used to build the software:
```
    CC       src/bzip3-main.o
    CC       src/libbz3.lo
    CCLD     libbzip3.la
    CCLD     bzip3
```

These are called "silent rules". In MacPorts we want those turned off so we can see exactly what was done.

```suggestion
                    size    399621
                 
configure.args      --disable-silent-rules
```

There was some discussion earlier in this PR about whether or not the build would use `-march=native`. Due to silent rules, I cannot see whether it did. Ports *must not* build with `-march=native` by default because the purpose of `-march=native` is to introduce CPU-specific optimizations. It would mean, for example, that a binary built on our build servers might not work on a user's system if they have a different (particularly an earlier) CPU. Ports *may* if desired introduce a "native" variant that uses `-march=native` or any other applicable CPU-specific optimizations.